cpp/src/ray/config_internal.cc

```diff
@@ -32,6 +32,8 @@ DEFINE_string(ray_logs_dir, "", "Logs dir for workers.");
 
 DEFINE_string(ray_node_ip_address, "", "The ip address for this node.");
 
+DEFINE_string(serialized_runtime_env, "{}", "The serialized parsed runtime env dict.");
```

Comment:
Why not "" as a default?

Reply:
It's serialized JSON, and "{}" is the value of the serialized empty dict.  `json.loads("")` would return an error.